index.js

```diff
@@ -7,7 +7,7 @@ module.exports = postcss.plugin('postcss-local-constants', function (opts) {
     var sets = opts && opts.defaults || {};
     var globalNode;
 
-    var regex = /((?:[\w]+))( )(from)(\s+)(~)((?:[A-z]+))/g;
+    var regex = /(~)((?:[\w]+))(\.)((?:[\w]+))/g;
```

Comment:
Having a look at this there seem to be a lot of redundant capture groups we’re not using?

I feel like `~([\w]+)\.([\w]+)` would do the same job, and allow the match indexes to just be `2` (for `constant`) and `1` (`constantSet`).